numpy/distutils/ccompiler_opt.py

```diff
@@ -721,9 +721,12 @@ def _dist_test_spawn(cmd, display=None):
         except subprocess.CalledProcessError as exc:
             o = exc.output
             s = exc.returncode
-        except OSError:
-            o = b''
-            s = 127
+        except OSError as e:
+            o = str(e)
```

Comment:
I'm curious why `o` is a `str` here, but bytes above.

Reply:
my bad, no need for it. already converted within string fmt.